core/tests/puppeteer-acceptance-tests/user-utilities/logged-in-users-utils.ts

```diff
@@ -40,6 +40,7 @@ const evenThoseWhoAreInSchoolUrl =
 const _420MillionUrl = testConstants.URLs.ExternalLink61MillionChildren;
 const thanksForDonatingUrl = testConstants.URLs.DonateWithThanksModal;
 const watchAVideoUrl = testConstants.URLs.ExternalLinkWatchAVideo;
```

Comment:
name this starting with "desktop", ditto in other cases you've added "mobile" to names where applicable

Reply:
Done